amundsen_application/static/js/components/ProfilePage/tests/index.spec.tsx

```diff
@@ -23,7 +23,7 @@ import {
 
 describe('ProfilePage', () => {
   const setup = (propOverrides?: Partial<ProfilePageProps>) => {
-    const props: ProfilePageProps = {
+    const props: Partial<ProfilePageProps> = {
```

Comment:
Since I added `RouteComponentProps` to the profile page the props object requires a bunch of nested objects. Is it worth using a shortcut or just defining everything in `RouteComponentProps` or just using a shortcut?

Reply:
There's some code that does this in `homepage/tests/index.spec.ts` but I feel like that's a bit onerous just to get some URL match params.

I'll leave this as is and possibly revisit later.